source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -158,7 +157,7 @@ The reason for using `const` is because the `$span` object is mutable. Whenever
 
 # Aliases
 
-Using `.then()` callback functions to access the previous command values is great - but what happens when you're running code in hooks like `before` or `beforeEach`?
+Using `.then()` callback functions to access the previous command values is great&mdash;but what happens when you're running code in hooks like `before` or `beforeEach`?
```

Comment:
Is there a style guide for the Cypress docs somewhere Jennfier? Even an informal one? I've replaced the ` - ` with an em-dash for style... yes, no?

> I'm not that fussed... the em-dash variant looks better to my eyes.